# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/android/SessionAccessibility.h
# Commit: 5956f91b3975
# Full Hash: 5956f91b3975ff848ab22b5b2abbd524422ede6d
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771934
# File Overlap Count: 1
# Description:
#   Bug 1771934 - Return null accessible when SessionAcc is not fully attached. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149295
# ==============================================================================

diff -r 8362b5d22558 -r 5956f91b3975 accessible/android/SessionAccessibility.h
--- a/accessible/android/SessionAccessibility.h	Tue Jun 28 18:56:34 2022 +0000
+++ b/accessible/android/SessionAccessibility.h	Tue Jun 28 19:40:08 2022 +0000
@@ -104,9 +104,7 @@
 
   void UpdateAccessibleFocusBoundaries(Accessible* aFirst, Accessible* aLast);
 
-  Accessible* GetAccessibleByID(int32_t aID) const {
-    return mIDToAccessibleMap.Get(aID);
-  }
+  Accessible* GetAccessibleByID(int32_t aID) const;
 
   static const int32_t kNoID = -1;
   static const int32_t kUnsetID = 0;
